models/stg_mailchimp.yml

```diff
@@ -475,12 +475,14 @@ models:
 
   - name: stg_mailchimp__members
     description: Each record represents a member in Mailchimp.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - member_id
+            - list_id
     columns:
       - name: member_id
```

Comment:
Shouldn't we still be testing on not_null?

Reply:
Ah true, added